packages/components/src/components/combobox-item/combobox-item.tsx

```diff
@@ -139,8 +141,14 @@ export class ComboboxItem extends LitElement {
    */
   @property() shortHeading: string;
 
-  /** The component's value. */
-  @property() value: any;
+  /** The component's value. Falls back to heading if not provided. */
```

Comment:
Awesome! 🤩  Can we wrap `heading` in back ticks for consistent formatting on the doc site?
```suggestion
  /** The component's value. Falls back to `heading` if not provided. */
```

Reply:
Updated! 🚀